shard/src/ext/pug_cog.py

```diff
@@ -0,0 +1,54 @@
+from discord.ext import commands
+from contextlib import suppress
+import time
+import asyncio
+import discord
+from lib.config import logger
+
+
+class Pug(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def startpug(self, ctx, role: discord.Role, requiredPlayers: int):
+        '''
+        Starts a tally for pugs
+        '''
+        try:
+            pug_emoji = discord.utils.get(ctx.guild.emojis, name="pugger")
```

Comment:
it might make more sense to use a unicode emoji here ✔️ or 👍 or something, just so we don't have a long list of custom emojis that servers have to have to use the features

Reply:
Yeah I just really hate emoji objects, I was kind of hoping you would point out some util that retrieves a PartialEmoji based off of a string so I could make this a configuration option.

Reply:
adding this to configuration options